src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar">
+      <div class="calendar_day calendar_day--1 start-day-tue"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--29 calendar--tue">
      <div class="calendar_day"></div>
```
Классы перенеси на обертку, а у каждого дня должен быть только один класс, calendar__day